pkg/irmaconfig.go

```diff
@@ -43,16 +43,9 @@ func GetIrmaConfig(config AuthConfig) (irmaConfig *irma.Configuration, err error
 			return
 		}
 
-		if !config.SkipAutoUpdateIrmaSchemas {
-			logrus.Infof("Downloading irma schemas. If this annoys you or you want to pin the schemas, set %s", ConfSkipAutoUpdateIrmaSchemas)
-			if err = irmaConfig.DownloadDefaultSchemes(); err != nil {
-				return
-			}
-		} else {
-			logrus.Info("Loading irma schemas.")
-			if err = irmaConfig.ParseFolder(); err != nil {
-				return
-			}
+		logrus.Info("Loading irma schemas.")
+		if err = irmaConfig.ParseFolder(); err != nil {
```

Comment:
DownloadDefaultSchemes is automatically called when auto-update = true?

Reply:
yup